### PR TITLE
core: frontend: utils: version_chooser: Deal if it fails to check bootstrap version

### PR DESCRIPTION
**Pull request by patrickelectric**

### CODE REVIEW

File: core/frontend/src/utils/version_chooser.ts [hunk 1]
```diff
@@ -153,12 +153,19 @@ async function loadCurrentVersion(): Promise<Version> {
...
   return back_axios({
     method: 'get',
     url: `${API_URL}/bootstrap/current/`,
     // eslint-disable-next-line no-extra-parens
   }).then((response) => (typeof response.data === 'object' ? undefined : response.data))
+    // The update process may fail and the user may be in bootstrap-backup
+    // This allows us to have the frontend working without crashing
+    // But still visible on the back
+    .catch((message) => {
+      console.warn(`Failed to fatch bootstrap version: ${message}`)
```
Comment (Williangalvani) [inline core/frontend/src/utils/version_chooser.ts:166]: I think this is worth of an alert/notification mentioning that their bootstrap might be bad.

Comment (patrickelectric) [reply core/frontend/src/utils/version_chooser.ts]: Done.

File: core/frontend/src/utils/version_chooser.ts [hunk 2]
```diff
@@ -153,12 +157,21 @@ async function loadCurrentVersion(): Promise<Version> {
...
     url: `${API_URL}/bootstrap/current/`,
     // eslint-disable-next-line no-extra-parens
   }).then((response) => (typeof response.data === 'object' ? undefined : response.data))
+    // The update process may fail and the user may be in bootstrap-backup
+    // This allows us to have the frontend working without crashing
+    // But still visible on the back
+    .catch((error) => {
+      const message = `Failed to fatch bootstrap version: ${error}`
+      notifier.pushWarning('VERSION_CHOOSER_FAILED_BOOTSTRAP_VERSION', message)
+      console.warn()
```
Comment (Williangalvani) [inline core/frontend/src/utils/version_chooser.ts:172]: does this empty warn() work? 🤔

Comment (patrickelectric) [reply core/frontend/src/utils/version_chooser.ts]: Ops.

**Review by Williangalvani — APPROVED** [review]

**Pull request merged**